app/policies/comment_policy.rb

```diff
@@ -0,0 +1,41 @@
+class CommentPolicy < ApplicationPolicy
+  attr_reader :user, :record
+
+  def index?
+    true
+  end
+
+  def show?
+    return true if beis_user?
+    user.organisation == record.report.organisation
+  end
+
+  def create?
+    return false if beis_user?
+    !editable_report_for_organisation.nil?
+  end
+
+  def edit?
+    update?
+  end
+
+  def update?
+    return false if beis_user?
+    editable_report_for_organisation_and_fund && record.owner.organisation == user.organisation
+  end
+
+  def destroy?
+    false
+  end
+
+  class Scope < Scope
```

Comment:
I don't think we use this scope and I would expect it to be different for BEIS users and Delivery partner users, maybe we should just loose it?